src/backend/base/langflow/components/composio/googlemeet_composio.py

```diff
@@ -0,0 +1,187 @@
+    name = "GooglemeetAPI"
+    icon = "Googlemeet"
+    documentation: str = "https://docs.composio.dev"
+    app_name = "googlemeet"
+
+    # Google Meet specific actions
+    _actions_data: dict = {
+        "GOOGLEMEET_GET_TRANSCRIPTS_BY_CONFERENCE_RECORD_ID": {
+            "display_name": "Get Transcripts By Conference Record ID",
+            "action_fields": ["GOOGLEMEET_GET_TRANSCRIPTS_BY_CONFERENCE_RECORD_ID-conferenceRecord_id"],
+        },
+        "GOOGLEMEET_GET_CONFERENCE_RECORD_FOR_MEET": {
+            "display_name": "Get Conference Record By Space Name, Meeting Code, Start Time, End Time",
+            "action_fields": [
+                "GOOGLEMEET_GET_CONFERENCE_RECORD_FOR_MEET-space_name",
+                "GOOGLEMEET_GET_CONFERENCE_RECORD_FOR_MEET-meeting_code",
+                "GOOGLEMEET_GET_CONFERENCE_RECORD_FOR_MEET-start_time",
+                "GOOGLEMEET_GET_CONFERENCE_RECORD_FOR_MEET-end_time",
+            ],
+        },
+        "GOOGLEMEET_GET_RECORDINGS_BY_CONFERENCE_RECORD_ID": {
+            "display_name": "Get Recordings By Conference Record ID",
+            "action_fields": ["GOOGLEMEET_GET_RECORDINGS_BY_CONFERENCE_RECORD_ID-conferenceRecord_id"],
+        },
+        "GOOGLEMEET_CREATE_MEET": {
+            "display_name": "Create Meet",
+            "action_fields": ["GOOGLEMEET_CREATE_MEET-access_type", "GOOGLEMEET_CREATE_MEET-entry_point_access"],
+        },
+        "GOOGLEMEET_GET_MEET": {
+            "display_name": "Get Meet Details",
+            "action_fields": ["GOOGLEMEET_GET_MEET-space_name"],
+        },
+    }
+
+    _all_fields = {field for action_data in _actions_data.values() for field in action_data["action_fields"]}
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self._default_tools = {
+            self.sanitize_action_name("GOOGLEMEET_CREATE_MEET").replace(" ", "-"),
+            self.sanitize_action_name("GOOGLEMEET_GET_MEET").replace(" ", "-"),
+        }
+        # Build the action maps right away
+        self._display_to_key_map = {data["display_name"]: key for key, data in self._actions_data.items()}
+        self._key_to_display_map = {key: data["display_name"] for key, data in self._actions_data.items()}
+        self._sanitized_names = {
+            action: self._name_sanitizer.sub("-", self.sanitize_action_name(action)) for action in self._actions_data
+        }
+
+    # Combine base inputs with Google Meet specific inputs
+    inputs = [
+        *ComposioBaseComponent._base_inputs,
+        MessageTextInput(
+            name="GOOGLEMEET_GET_TRANSCRIPTS_BY_CONFERENCE_RECORD_ID-conferenceRecord_id",
+            display_name="Conference Record Id",
+            info="The unique identifier for the conference record.",
+            show=False,
+            required=True,
+        ),
+        MessageTextInput(
+            name="GOOGLEMEET_GET_RECORDINGS_BY_CONFERENCE_RECORD_ID-conferenceRecord_id",
```

Comment:
The name of the input becomes an attribute so it has to be a variable compliant name.